client/src/components/navigations/RootStackNavigator.tsx

```diff
@@ -200,7 +201,9 @@ const RootNavigatorWrapper: FC = () => {
       }}>
       <Suspense fallback={<CustomLoadingIndicator />}>
         {meQueryReference ? (
-          <RootNavigator queryReference={meQueryReference} />
+          <SnackbarProvider>
+            <RootNavigator queryReference={meQueryReference} />
+          </SnackbarProvider>
```

Comment:
I think it will be more consistent if we could put `SnackbarProvider` in `src/App.tsx` file (see line 100).

Reply:
I think so, too!
i'll put this component in`src/App.js`